source/Core/InputValidator.php

```diff
@@ -174,37 +176,47 @@ public function checkEmail($user, $email)
      * Checking if user password is fine. In case of error
      * exception is thrown
      *
-     * @param User   $user                      Active user.
+     * @param string $email                     Email.
      * @param string $newPassword               New user password.
      * @param string $confirmationPassword      Retyped user password.
      * @param bool   $shouldCheckPasswordLength Option to check password length.
      *
      * @return Exception\StandardException|null
      */
-    public function checkPassword($user, $newPassword, $confirmationPassword, $shouldCheckPasswordLength = false)
+    public function checkPassword($email, string $newPassword, string $confirmationPassword, bool $shouldCheckPasswordLength = false)
```

Comment:
As we can not change, what we pass, we should also leave the argument name as `$user`